src/private_broadcast.cpp

```diff
@@ -81,6 +81,19 @@ bool PrivateBroadcast::FinishBroadcast(const NodeId& nodeid, bool confirmed_by_n
     return true;
 }
 
+std::vector<CTransactionRef> PrivateBroadcast::GetStale() const EXCLUSIVE_LOCKS_REQUIRED(!m_mutex)
+{
+    LOCK(m_mutex);
+    const auto stale_time = GetTime<std::chrono::microseconds>() - 1min;
```

Comment:
Yes, moved to a constant and added a comment to it. Placed the constant in the `.cpp` file since it is only used in that `.cpp` file, to limit its scope. It is not part of the "public" `private_broadcast.h/cpp` interface